test/index.test.ts

```diff
@@ -0,0 +1,104 @@
+import connect from 'connect';
+import request from 'supertest';
+import { IncomingMessage, ServerResponse } from 'http';
+
+import clearSiteData = require('..');
+
+type Directive = 'cache' | 'cookies' | 'storage' | 'executionContexts' | '*';
+
+const WHITELIST: Directive[] = [
```

Comment:
What happens if you specify this as an array of strings instead of `Directive`s? I worry that a user of this module will have to do some extra legwork to use "regular" strings here.